lib/explorer/data_frame.ex

```diff
@@ -5648,23 +5652,30 @@ defmodule Explorer.DataFrame do
       raise ArgumentError, "cannot describe a DataFrame without any columns"
     end
 
-    stat_cols = for {name, type} <- df.dtypes, type in Shared.numeric_types(), do: name
+    stat_cols = df.names
     percentiles = process_percentiles(opts[:percentiles])
+    numeric_types = Shared.numeric_types()
+    datetime_types = Shared.datetime_types()
+    duration_types = Shared.duration_types()
 
     metrics_df =
       summarise_with(df, fn x ->
         Enum.flat_map(stat_cols, fn c ->
+          dt = x[c].dtype
+          numeric? = dt in numeric_types
+          min_max? = numeric? || dt in datetime_types || dt in duration_types
```

Comment:
```suggestion
          min_max? = numeric? or dt in datetime_types or dt in duration_types
```

Since they are all guaranteed to be boolean, using `or` is clearer :)